app/serializers/rest/account_serializer.rb

```diff
@@ -114,6 +124,10 @@ def silenced
     object.silenced?
   end
 
+  def role
```

Comment:
Could we name this `roles` and make it an array to future-proof for possible changes in the future?

Reply:
done